backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -71,12 +70,11 @@ public UsersController(WebClientService webClientService, @Value("${config.vanit
     }
 
     @GetMapping("/users/{userId}/role-mappings/clients/{clientGuid}")
-    public Mono<Object> getAssignedUserClientRoleMapping(
+    public ResponseEntity<Object> getAssignedUserClientRoleMapping(
             @RequestHeader("Authorization") String token,
             @PathVariable String userId,
             @PathVariable String clientGuid) {
 
-        //Check if the user has the required role to view information related to this client
```

Comment:
I removed this comment because it's the same comment on every method invocation, which means it could go in the method name or the method Javadoc. I actually think `isAuthorizedToViewClient` is clear enough though; I don't think "Check if the user has the required role to view information related to this client" adds anything.